### ISSUE TITLE
clem_mipmapper automatically inverts contrast

### ISSUE THREAD

**Issue by arentkievits** [hoogenboom-group/scripted-render-pipeline#5]
`def` create_mipmap_from_page(
        self,
        page,
        x_by_y,
        root,
        image_elements_by_name,
        detector_by_id,
        datatype_dir,
        file_path,
        section_name,
        zvalue,
    ):
        """create mipmaps for a single page in the multipage tiffile

        page: tifffile.TiffPage to interpret
        x_by_y: x and y count as tuple
        root: metadata from this tifffile
        image_elements_by_name: dictionary of metadata for image elements
        detector_by_id: dictionary of detector names for ids
        datatype_dir: type of capture
        file_path: path of this tifffile
        section_name: name of this stack
        zvalue: z height of this section
        """
        channel = page.tags["PageName"].value
        element = image_elements_by_name[channel]
        new_root = copy.copy(root)
        for other in image_elements_by_name.values():
            if other != element:
                new_root.remove(other)

        description = xml.etree.ElementTree.tostring(
            new_root,
            encoding="unicode",
            xml_declaration=False,
        )
        # tifffile.OmeXml.validate(description)
        image = page.asarray()
        pixels = element.find("Pixels", NAMESPACE)
        if channel == "Secondary electrons":
            name = DIR_BY_DATATYPE[datatype_dir]
            image = skimage.util.invert(image)  # invert the SEM image
            intensity_clip = 1, 99
`

**Comment by thopp-tudelft** [comment]
the inversion is done because it was also done in the original workflow, it would be fairly easy to make this into an optional parameter

**Comment by thopp-tudelft** [comment]
would it be possible to detect automatically when inversion is needed?

**Comment by arentkievits** [comment]
I don't think so. It is determined by the detector settings, which are not saved in the metadata. Would be non-trival to infer the contrast type from the image content. 

For CLEM data from our microscope, the contrast inversion would best be a parameter with default value "True", but can be set to "False".

**Comment by thopp-tudelft** [comment]
reading the comments here I don't think we'll implement automatic contrast inversion, the implementation in #33 with the parameter is sufficient
